tests/time_tests/scripts/run_timetest.py

```diff
@@ -154,13 +154,15 @@ def cli_parser():
     logging.basicConfig(format="[ %(levelname)s ] %(message)s",
                         level=logging.DEBUG, stream=sys.stdout)
 
-    exit_code, aggr_stats = run_timetest(dict(args._get_kwargs()), log=logging)  # pylint: disable=protected-access
+    exit_code, aggr_stats, stats = run_timetest(dict(args._get_kwargs()), log=logging)  # pylint: disable=protected-access
 
     if args.stats_path:
-        # Save aggregated results to a file
+        # Save all results to a file
+        for step_name in aggr_stats:
+            aggr_stats[step_name].update({"vals": stats[step_name]})
```

Comment:
I'm argue with updating `aggr_stats` if it's required for logging only. Let's create separate variable. It may be done via `{step_name: {**aggr_stats[step_name], **stats[step_name]} for step_name in aggr_stats}` directly in 163 line (after `with` statement)

Reply:
Anyway, since we want to save raw results in DB, I think we may skip saving raw results in a file if script is run via command-line